rust_src/src/buffers.rs

```diff
@@ -428,4 +428,15 @@ pub fn barf_if_buffer_read_only(position: Option<EmacsInt>) -> () {
     }
 }
 
+/// No such buffer error.
+#[no_mangle]
+pub extern "C" fn nsberror(spec: Lisp_Object) -> ! {
+    let spec = LispObject::from_raw(spec);
+    if let Some(s) = spec.as_string() {
+        error!("No buffer named {}", s);
```

Comment:
Yes, this calls fromtutf8_lossy behind the scenes. But frankly we have similar issues elsewhere so I see no reason to hold up the PR for it.